lib/bolt/cli.rb

```diff
@@ -447,6 +448,7 @@ def show_task(task_name)
     def list_tasks
       tasks = pal.list_tasks
       tasks.select! { |task| task.first.include?(options[:filter]) } if options[:filter]
+      tasks.select! { |task| config.project.tasks.include?(task.first) } if config.project.tasks.any?
```

Comment:
This means there's no way\* to have a project that only includes tasks _or_ plans but not both, ie. there's no way to specify that you _want_ a whitelist but that it's empty.

\* There technically is a hacky way which is to make a whitelist that contains a task or plan that doesn't actually exist

Reply:
Setting this to `tasks: []` should correctly show no tasks in `bolt task show`, while `tasks: ` throws an error.